tests/fixtures/create/create.json

```diff
@@ -1,5 +1,5 @@
 {
   "type": "create",
-  "suffix_data": "eyJkZWx0YV9oYXNoIjoiRWlCWE00b3RMdVAyZkc0WkE3NS1hbnJrV1ZYMDYzN3hadE1KU29Lb3AtdHJkdyIsInJlY292ZXJ5X2NvbW1pdG1lbnQiOiJFaUM4RzRJZGJEN0Q0Q281N0dqTE5LaG1ERWFicnprTzF3c0tFOU1RZVV2T2d3In0",
-  "delta": "eyJ1cGRhdGVfY29tbWl0bWVudCI6IkVpQ0lQY1hCempqUWFKVUljUjUyZXVJMHJJWHpoTlpfTWxqc0tLOXp4WFR5cVEiLCJwYXRjaGVzIjpbeyJhY3Rpb24iOiJyZXBsYWNlIiwiZG9jdW1lbnQiOnsicHVibGljX2tleXMiOlt7ImlkIjoic2lnbmluZ0tleSIsInR5cGUiOiJFY2RzYVNlY3AyNTZrMVZlcmlmaWNhdGlvbktleTIwMTkiLCJqd2siOnsia3R5IjoiRUMiLCJjcnYiOiJzZWNwMjU2azEiLCJ4IjoieTlrenJWQnFYeDI0c1ZNRVFRazRDZS0wYnFaMWk1VHd4bGxXQ2t6QTd3VSIsInkiOiJjMkpIeFFxVVV0eVdJTEFJaWNtcEJHQzQ3UGdtSlQ0NjV0UG9jRzJxMThrIn0sInB1cnBvc2UiOlsiYXV0aCIsImdlbmVyYWwiXX1dLCJzZXJ2aWNlX2VuZHBvaW50cyI6W3siaWQiOiJzZXJ2aWNlRW5kcG9pbnRJZDEyMyIsInR5cGUiOiJzb21lVHlwZSIsImVuZHBvaW50IjoiaHR0cHM6Ly93d3cudXJsLmNvbSJ9XX19XX0"
+  "suffix_data": "eyJkZWx0YV9oYXNoIjoiRWlDTW5jb0xrODlIeWpUYzhRbUU0VVVoNFpzMlhiOGlGTWxBak51WFlDbjZCdyIsInJlY292ZXJ5X2NvbW1pdG1lbnQiOiJFaUF5VklubUZfbnpwYWVmeVdqNkJvTkdkWV85RlJOZWYtV0kzY3JySzZ2SXlBIn0",
```

Comment:
Was there a corresponding change to the spec that has resulted in the fixtures changing?

Reply:
This is part of SIP1, the thing where we need to remove the revealed public key JWK value from the Anchor File, which requires hashing it twice, do you don't have to put the actual data there.

Reply:
Adding to @csuwildcat, this does impact the fixture/vector because with this PR commit value needs to be double hash of public key as opposed to single hash.